test/browser/component.js

```diff
@@ -316,6 +316,20 @@ describe(`Nested Component instance`, function() {
       childEl.update({animal: `capybara`});
       expect(el.state.animal).to.equal(`capybara`);
     });
+
+    it.only(`flushes child state updates to parent`, async function() {
```

Comment:
```suggestion
    it(`flushes child state updates to parent`, async function() {
```

Reply:
oh my 🤦‍♂ need to do something about that